includes/class-support-snapshot.php

```diff
@@ -46,7 +46,7 @@ public static function ping_relay() {
 		if ( isset( $_REQUEST['np_dispatch'] ) && isset( $_REQUEST['action' ] ) && 'trigger_snapshot' == $_REQUEST['action'] ) {
 			$options          = get_option( 'blog_tutor_support_settings' );
 			$site_api_key     = $options['np_relay_api_token'] ?? '';
-			$relay_server_url = ! empty( $options['np_relay_server_url'] ) ? $options['np_relay_server_url'] : 'https://relay.nerdpress.net';
+			$relay_server_url = ! empty( $options['np_relay_server_url'] ) ? $options['np_relay_server_url'] : NerdPress_Helpers::relay_server_default_url();
```

Comment:
```suggestion
			$relay_server_url = ! empty( $options['np_relay_server_url'] ) ? $options['np_relay_server_url'] : rtrim( NerdPress_Helpers::relay_server_default_url(), '/' );
```

I sent  you a message about this, but we want to keep that trailing slash removed here so everything will behave exactly the same.